pkgs/development/libraries/itk/default.nix

```diff
@@ -23,12 +24,17 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  nativeBuildInputs = [ cmake xz ];
-  buildInputs = [ libX11 libuuid vtk ] ++ stdenv.lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Cocoa ];
+  nativeBuildInputs = [ cmake xz makeWrapper ];
+  buildInputs = [ libX11 libuuid vtk ] ++ stdenv.lib.optionals stdenv.isDarwin [ Cocoa ];
+  propagatedBuildInputs = [ pkgconfig ];
```

Comment:
I guess this one is not needed anymore. It also should have been `propagatedNativeBuildInputs`.